engine/src/main/java/org/terasology/physics/shapes/CompoundShape.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.physics.shapes;
+
+import org.terasology.math.Transform;
+
+/**
+ * Represents a compound collision entity composed of other {@link CollisionShape}s in the physics engine.
```

Comment:
I'd remove "in the physics engine" - it's clear enough without it.

Reply:
Ok, after a comment above I better understand what you mean now. We need to disambiguate the ambiguity though. Despite the three repetitions of the word "shape" I'd write:

`* Represents a compound collision shape, a shape composed of other {@link CollisionShape}s, within the physics engine. `

Reply:
The alternative would be to keep the term "entity", but then we'd have to clarify with a note that we are referring to entities within the physics engine rather Terasology's entities.